ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly.yaml

```diff
@@ -605,6 +605,42 @@ tests:
     - ref: idp-htpasswd
     - ref: openshift-extended-test-longduration
     workflow: cucushift-installer-rehearse-aws-ipi-ovn
+- as: aws-ipi-ovn-hypershift-mce-ibmz-guest-critical-f7
+  cron: 3 10 6,13,20,27 * *
+  steps:
+    cluster_profile: aws-qe
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      E2E_RUN_TAGS: '@amd64 and @ibmcloud-ipi and @network-ovnkubernetes and @critical
+        and @hypershift-hosted and not @fips'
+      ISO_URL: https://mirror.openshift.com/pub/openshift-v4/s390x/dependencies/rhcos/pre-release/4.14.0-ec.2/rhcos-4.14.0-ec.2-s390x-live.s390x.iso
+      MCE_VERSION: "2.4"
+      OCP_RELEASE_VERSION: 414.92.202306100411-0
+      OCP_VERSION: 4.14.0-ec.2
```

Comment:
We have several release version for 4.14. What's the reason we are using ec.2 here ?

Reply:
Here I follow ibm QE's config.  Do you know how to config it with the nightly version dynamically ? thanks

Reply:
For `xxx__amd64-nightly.yaml`, the OCP version comes from `release:latest`, and stored in `RELEASE_IMAGE_LATEST`